doc/en/how-to/index.rst

```diff
@@ -3,19 +3,58 @@
 How-to guides
 ================
 
+Core pytest functionality
+-------------------------
+
 .. toctree::
    :maxdepth: 1
 
    usage
-   existingtestsuite
    assert
+   fixtures
    mark
-   monkeypatch
+   parametrize
    tmpdir
-   capture
+   monkeypatch
+   doctest
+   cache
+
+pytest output and outcomes
+----------------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   logging
+   capture-stdout-stderr
+   capture-warnings
    skipping
-   parametrize
+
+Plugins
+----------------------------
+
+.. toctree::
+   :maxdepth: 1
+
    plugins
+   writing_plugins
+   writing_hook_functions
+
+pytest and other test systems
+-----------------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   existingtestsuite
    nose
```

Comment:
If the order is meaningful, I would move nose below unittest, as unittest is much more common.